openwisp_radius/base/models.py

```diff
@@ -1055,15 +1073,38 @@ def clean(self):
                     )
                 }
             )
+        # Clean `freeradius_allowed_hosts`
+        ip_list = self.freeradius_allowed_hosts_list
```

Comment:
I will suggest you to rename `ip_list` to `allowed_hosts` for better readability. What do you think @atb00ker ?